streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -490,6 +494,19 @@ abstract class DStream[T: ClassTag] (
     logDebug("Cleared checkpoint data")
   }
 
+  private[streaming] def readyToShutdown(): Unit = {
+    _readyToShutdown = true
+    dependencies.foreach(_.readyToShutdown())
+    logDebug("Ready to shutdown")
```

Comment:
yes I know, I mean you have one https://github.com/apache/spark/pull/11101/files#diff-221bc6301915f7a476786c794b855b21R105 already which print out exactly the same log, probably we can add more information in this log? for example: creationSite?